dev-lang/rust/rust-9999.ebuild

```diff
@@ -303,17 +324,15 @@ pkg_setup() {
 src_prepare() {
 	if [[ ${PV} = *9999* ]]; then
 		# We need to update / generate lockfiles for the workspace
-		${CARGO} generate-lockfile --offline
-
-	fi
-	# Rust baselines to Pentium4 on x86, this patch lowers the baseline to i586 when sse2 is not set.
-	if use x86; then
-		if ! use cpu_flags_x86_sse2; then
-			eapply "${FILESDIR}/1.82.0-i586-baseline.patch"
-			#grep -rl cmd.args.push\(\"-march=i686\" . | xargs sed  -i 's/march=i686/-march=i586/g' || die
-		fi
+		${CARGO} generate-lockfile --offline || die "Failed to generate lockfiles"
 	fi
 
+	shopt -s nullglob
+	PATCHES=(
+		"${WORKDIR}/rust-patches-${RUST_PATCH_VER}/"*.patch
```

Comment:
You don't need to do this dance - PATCHES when given a directory will do exactly that, i.e. drop the shopt + *.patch.

Reply:
This doesn't seem resolved?

Reply:
I haven't pushed my changes yet, just checking things off!